main.cpp

```diff
@@ -114,155 +114,249 @@ struct CarWash //                                   1) define an empty struct fo
 };
 
 
-/*
-1)
-Skateboard
-5 properties:
-    1) Grip tape color
-    2) Hardness of Wheels
-    3) Brand of Trucks
-    4) Age of Bearings
-    5) Type of Wood
-3 things it can do:
-    1) roll
-    2) slide
-    3) flip
- */
 
-/*
-2) Car
-5 properties:	
-    1) Brand of Radio	    
-    2) Number of Tires	    
-    3) Size of Engine	    
-    4) Brand of Rims	    
-    5) Count of Seats	    
-3 things it can do:	
-    1) Drive	    
-    2) Park	   
-    3) Stop
+//1)
+struct Skateboard
+{//5 properties:
+    //1) Grip tape color
+    std::string gripTapeColor = "Green";
+    //2) Hardness of Wheels
+    int wheelsHardness = 99;
+    //3) Brand of Trucks
+    std::string truckBrand = "Independent";
+    //4) Age of Bearings
+    int bearingAgeYears = 2;
+    //5) Type of Wood
+    std::string woodType = "oak";
+//3 things it can do:
+    //1) roll
+    void rollDownStreet(int speed());
+    //2) slide
+    void railSlide(bool frontSide = true);
+    //3) flip
+    void kickFlip(int degree = 360);
+};
 
- */
 
-/*
-3)
-Computer
-5 properties:
-    1) Feel of keys
-    2) Resolution of Monitor
-    3) Kinds of indictor lights
-    4) Number of buttons
-    5) Speed of Processor
-3 things it can do:
-    1) do taxes
-    2) play music
-    3) display numbers
- */
+//2) 
+struct Car
+{
+//5 properties:	
+    //1) Brand of Radio	    
+    std::string radioBrand = "Sony";
+    //2) Number of Tires	    
+    int numTires = 4;
+    //3) Size of Engine	    
+    int engineSize = 4600;
+    //4) Brand of Rims	    
+    std::string rimBrand = "MSR";
+    //5) Count of Seats
+    int seatCount = 2;	    
+//3 things it can do:	
+    //1) Drive	
+    void drive(int speed());    
+    //2) Park
+    void park(int parkingStallNumber());	   
+    //3) Stop
+    void stop(int stopDistance());
```

Comment:
your function parameters are not functions themselves.  While you can pass functions as parameters if you declare correctly, you are not there yet. 
- [ ] revise.  remove the `()` after your function parameter's name.